README.md

```diff
@@ -1,80 +1,61 @@
 # Brands Reader/Writer for Neo4j (brands-rw-neo4j)
 
-__An API for reading/writing brands into Neo4j. Expects the brands json supplied to be in the format that comes out of the brands transformer.__
+__An API for reading/writing brands into Neo4j. Expects the brands json supplied to be in the format that comes out of the brands extractor.__
 
-## Installation
-
-For the first time:
-
-`go get github.com/Financial-Times/brands-rw-neo4j`
-
-or update:
+[Runbook for service](https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/draftbrandsreadwritemicroservicedanmurphyftcom20160215)
 
```

Comment:
You can (and should) amend the url for the run book. I think it's in Manage Site for that page on the google site.